app/src/main/res/layout/item_content_cart.xml

```diff
@@ -77,6 +77,8 @@
                             android:layout_width="wrap_content"
                             android:layout_height="wrap_content"
                             android:gravity="center"
+                            android:paddingEnd="@dimen/dp_5"
```

Comment:
WARNING: [RtlSymmetry] When you define `paddingEnd` you should probably also define `paddingStart` for right-to-left symmetry
 If you specify padding or margin on the left side of a layout, you should probably also specify padding on the right side (and vice versa) for right-to-left layout symmetry.